examples/scripts/index.rst

```diff
@@ -159,3 +159,14 @@ PCG
 ^^^
 
    - ct_astra_noreg_pcg.py
+
+
```

Comment:
Not clear why the `Machine Learning` section under `Organized by Application` that appears in the RTD docs doesn't appear here, but I would recommend moving that section to a subsection of `Organized by Regularization`.